robotoff/app/api.py

```diff
@@ -458,22 +468,19 @@ def on_post(self, req: falcon.Request, resp: falcon.Response):
         server_domain = req.get_param("server_domain", required=True)
 
         if server_domain != settings.OFF_SERVER_DOMAIN:
-            logger.info("Rejecting image import from {}".format(server_domain))
+            logger.info(f"Rejecting image import from {server_domain}")
             resp.media = {
                 "status": "rejected",
             }
             return
 
-        send_ipc_event(
-            "import_image",
-            {
-                "barcode": barcode,
-                "image_url": image_url,
-                "ocr_url": ocr_url,
-                "server_domain": server_domain,
-            },
+        high_queue.enqueue(
+            run_import_image_job,
+            barcode=barcode,
+            image_url=image_url,
+            ocr_url=ocr_url,
+            server_domain=server_domain,
```

Comment:
shouldn't you use enqueue_job to be consistent ?

Reply:
yes totally!